app/controllers/application_controller/tree_support.rb

```diff
@@ -20,6 +20,11 @@ def squash_toggle
     end
   end
 
+  def find_record
+    identify_record(params[:id], controller_name.camelize) if %w(container_image host).include? controller_name
+    identify_record(params[:id], VmOrTemplate)
```

Comment:
> I wish we could do model.base_class, but I don't really know why we can't.
Is there a model available? Or should that be acquired from the controller name? I could submit another patch for that.